src/ignitus-Api/index.js

```diff
@@ -1,8 +1,4 @@
-
-
 import axios from 'axios';
-import * as t from './constants';
+import TESTIMONIAL_URL from './constants';
```

Comment:
Check my above comment for the same. We may have more constants here.

Reply:
yes, please ignore default.

Reply:
okay